utils/choices.py

```diff
@@ -1,9 +1,70 @@
-from django.db.models import IntegerChoices
+from django.db.models import TextChoices, IntegerChoices
 
 class ExampleChoices(IntegerChoices):
     '''
     예시 코드입니다.
     '''
     ONE = 1, '일'
     TWO = 2, '이'
-    THREE = 3, '삼'
\ No newline at end of file
+    THREE = 3, '삼'
+
+class FounderTargetChoices(TextChoices):
+    LOCAL = "local", "동네주민"
+    OUTSIDER = "outsider", "외부인"
```

Comment:
💊 토스페이먼츠에서 대문자만 사용하던데, 우리도 그렇게 하면 어떨까요?
```suggestion
class FounderTargetChoices(TextChoices):
    LOCAL = "LOCAL", "동네주민"
    OUTSIDER = "OUTSIDER", "외부인"
```